### PR TITLE
CR-1087744 ASTeR basic sanity - U30 - power_warn is non zero

### PR DESCRIPTION
**Pull request by xuhz**
CR-1080204 U30 AWS: Make a read/write interface to the QSPI in a PCIe BAR

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 1]
```diff
@@ -733,7 +734,10 @@ static void xmc_sensor(struct platform_device *pdev, enum data_kind kind,
 			READ_SENSOR(xmc, XMC_VCCRAM_REG, val, val_kind);
 			break;
 		case XMC_POWER_WARN:
-			READ_SENSOR(xmc, XMC_POWER_WARN_REG, val, val_kind);
+			safe_read32(xmc, XMC_POWER_WARN_REG, val);
```
Comment (xdavidz) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c:737]: read32 without val_kind means read offset+0 (SENSOR_MAX). Are you sure that is the value we are reading? I am confuzed.

Comment (xuhz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c:737]: that one doesn't have max/min/ins defined. only offset 0 is available and it is the one they want

**Review by xdavidz — APPROVED** [review]

**Comment by xilinxgitops** [comment]
retest this please

**Comment by xuhz** [comment]
retest this please

**Pull request merged**